examples/simple.rs

```diff
@@ -1,40 +1,65 @@
-
 extern crate rspec;
-use rspec::context::describe;
+
 use std::io;
+use std::sync::{Arc, Mutex};
+use std::collections::BTreeSet;
 
-pub fn main() {
-    let stdout = &mut io::stdout();
-    let mut formatter = rspec::formatter::Simple::new(stdout);
-    let mut runner = describe("rspec is a classic BDD testing", |ctx| {
+use rspec::prelude::*;
 
-        ctx.it("can define tests", || true);
+pub fn main() {
```

Comment:
keep in mind examples are meant to be read. This is not readable by a beginner. Please comment as you saw this file for the first time.

What question would ask you a newbie ?